Foundation/URLCache.swift

```diff
@@ -45,11 +45,42 @@ extension URLCache {
 open class CachedURLResponse : NSObject, NSSecureCoding, NSCopying {
     
     public required init?(coder aDecoder: NSCoder) {
-        NSUnimplemented()
+        guard aDecoder.allowsKeyedCoding else {
+            /* Unkeyed unarchiving is not supported. */
+            return nil
+        }
+
+        guard let data = aDecoder.decodeObject(of: NSData.self, forKey: "Data") else {
```

Comment:
Can you remove the coding-related parts of this patch? The coding testing isn't done like this anymore (see `FixtureValues.swift` and grep the codebase for `codingRoundtrip`) and this isn't compatible with Darwin's encoding.